core/src/main/scala/cromwell/core/io/IoCommand.scala

```diff
@@ -28,12 +28,12 @@ trait IoCommand[+T] {
     * Completes the command successfully
     * @return a message to be sent back to the sender, if needed
     */
-  def success[S >: T](value: S): IoSuccess[S] = IoSuccess(this, value)
+  def success[TT >: T](value: TT): IoSuccess[TT] = IoSuccess(this, value)
```

Comment:
Curious, why the anti-`S` bias in this file? IMHO if we're making changes to this file, the trait should switch to `IoCommand[+A]`, and this should should go to `B >: A`.

Reply:
Ah yes, that was a leftover from an earlier version. I'll revert